src/jdk.compiler/share/classes/com/sun/tools/javac/comp/TransPatterns.java

```diff
@@ -65,13 +66,8 @@
 import com.sun.tools.javac.util.Name;
 import com.sun.tools.javac.util.Names;
 
-import java.util.Collections;
-import java.util.Map;
+import java.util.*;
```

Comment:
nit: I think we prefer not using star imports, this was probably done by the IDE